slidey_js.js

```diff
@@ -0,0 +1,36 @@
+$(document).ready(function(){
+
+backgroundImage = $("#background")
+
+images = [
+  "url('http://i.giphy.com/e34HHm5F08FqM.gif')",
+  "url('http://i.giphy.com/JzIyZB0fY4dPO.gif')",
+  "url('http://i.giphy.com/P62H3xNZL24ow.gif')"
+]
+
+imageArray = images[0]
+
+backgroundImage.addEventListener("loadup", backgroundInterval())
+
+
+function backgroundInterval(){
+  window.setInterval(startup, 5000)
+};
+
+function startup(){
+  // backgroundImage.css({
+  //   "background-image": "url('#{images[0] + images}')";
+  // })
+
+  for(i in images){
```

Comment:
Try using a `for(var i = 0; i < length; i++)` loop when looping through items of a known length